### PR TITLE
Bump flipper gems to latest

### PR DESCRIPTION
**Pull request by LindseySaari**
## Description of change
As part of the gem update series, this PR updates the flipper gem and associated gems(default group) to the latest minor version. Gems are being updated individually (separate PRs), in order to avoid issues if a roll back is needed.

https://github.com/jnunemaker/flipper/blob/master/Changelog.md

The VAOS tests that were failing were due to a change in how Statsd gets initalized. The issue pertains to the new version of flipper along with StatsD.backend being set to StatsD::Instrument::Backends::NullBackend whereas before the flipper update, Statsd.backend was set to StatsD::Instrument::Backends::LoggerBackend (which is a wrapper for the Rails logger). When Statsd.backend is set to StatsD::Instrument::Backends::NullBackend, the Rails logger is never called causing the test to fail. The VAOS team has skipped (xit) the failing specs and will address at a later time. Additionally, the team did some testing by setting RAILS_ENV to development, which sets Statsd.backend to StatsD::Instrument::Backends::LoggerBackend and the tests pass. 

## Original issue(s)
department-of-veterans-affairs/va.gov-team##24724

## Testing Completed
- [x] CI make target passing locally 
- [x] Reviewed gem change log updates 
- [x] Tested flipper UI locally

### CODE REVIEW

**Comment by LindseySaari** [comment]
Next steps: test flipper-ui and functionality

File: config/initializers/flipper.rb [hunk 1]
```diff
@@ -10,6 +10,12 @@
 
 FLIPPER_FEATURE_CONFIG = YAML.safe_load(File.read(Rails.root.join('config', 'features.yml')))
 
+# config/initializers/flipper.rb
```
Comment (ericboehs) [inline config/initializers/flipper.rb:13]:
```suggestion
```

File: config/initializers/flipper.rb [hunk 2]
```diff
@@ -10,6 +10,12 @@
...
+Rails.application.configure do
+  # Or completely disable preloading
+  config.flipper.memoize = false
```
Comment (ericboehs) [inline config/initializers/flipper.rb:16]: You should probably do this in the `Flipper.configure` block below.

**Comment by LindseySaari** [comment]
@ericboehs This should be good to go!

File: config/application.rb [hunk 3]
```diff
@@ -66,6 +66,10 @@ class Application < Rails::Application
       end
     end
 
+    # combats the Flipper::Middleware::Memoizer appears to be running twice error
```
Comment (ericboehs) [inline config/application.rb:69]:
```suggestion
    # combats the "Flipper::Middleware::Memoizer appears to be running twice" error
```

File: modules/check_in/README.rdoc [hunk 4]
```diff
@@ -1,10 +1,15 @@
-= CheckIn
-TODO: Short description and motivation.
+# What is vets-api CheckIn?
```
Comment (ericboehs) [inline modules/check_in/README.rdoc:1]: Is this file related to the flipper change?

**Review by ericboehs — COMMENTED** [review]
Test comment

File: modules/check_in/README.rdoc (hunk 4 above)
Comment (LindseySaari) [reply modules/check_in/README.rdoc:1]: This must have been a merge conflict

**Review by ericboehs — APPROVED** [review]
